.github/workflows/gcc.yml

```diff
@@ -35,6 +35,8 @@ jobs:
       run: make -j`nproc` -C import-chrome/build/unix debug release
     - name: Library
       run: make -j`nproc` -C library/unix debug release
+    - name: Library (meson)
+      run: meson setup build && meson compile -C build
```

Comment:
This probably needs to be excluded on macos.

Reply:
Why so? Appears to have worked: https://github.com/YaLTeR/tracy/actions/runs/6572890389

Reply:
Never mind then. Maybe we should also do `ninja install` and then publish the artifacts?

Reply:
I thought about this a bit, especially considering that I do package this .so for my instrumented mutter repo. However, I don't think it makes much sense to do generally, because in practice you do want to set different defines based on the use-case (i.e. I set `TRACY_ONLY_LOCALHOST` for anything I send to others, yet I leave it off for my own machines), and a built library supports only a single set of defines. If it was configured at runtime instead, then it would make more sense to publish it IMO.

Reply:
I guess `TRACY_ONLY_LOCALHOST` is not a good example, `TRACY_MANUAL_LIFETIME` is a better one that needs explicit support from the profiled application.

Reply:
> If it was configured at runtime instead, then it would make more sense to publish it IMO.

My intention was more to see what the install job does, if all the files are where they should be, etc. It may not be apparent just from looking at the build script.

Reply:
Hm, then could do an install with a prefix set to local dir. Meson prints all files it installs.

Reply:
Sure.

Reply:
https://github.com/YaLTeR/tracy/actions/runs/6573262386